### PR TITLE
Rename model and variables

### PR DESCRIPTION
**Pull request by damonbayer**
Closes https://github.com/CDCgov/pyrenew-hew/issues/177

### CODE REVIEW

**Comment by damonbayer** [comment]
@dylanhmorris ready for review

File: demos/hosp_only_ww_model/wwinference.Rmd [hunk 1]
```diff
@@ -280,7 +280,7 @@ modified with any discrete daily PMF.
 
 ```{r set-delay-distributions}
 generation_interval <- wwinference::default_covid_gi
-inf_to_hosp <- wwinference::default_covid_inf_to_hosp
+inf_to_ed <- wwinference::default_covid_inf_to_ed
```
Comment (dylanhmorris) [inline demos/hosp_only_ww_model/wwinference.Rmd:283]: `wwinferrence::default_covid_inf_to_ed` doesn't exist. Just use inf to hosp and explain why, then later update this tutorial to use PyrenewH

**Review by dylanhmorris — CHANGES_REQUESTED** [review]
One tiny but very important change, otherwise looks good.

File: demos/hosp_only_ww_model/wwinference.Rmd (hunk 1 above)
Comment (damonbayer) [reply demos/hosp_only_ww_model/wwinference.Rmd]: Resolved in https://github.com/CDCgov/pyrenew-hew/pull/186/commits/fb8e504de519bf00acabb2acd56b12d87bf38d57

Comment (damonbayer) [reply demos/hosp_only_ww_model/wwinference.Rmd]: Would have been caught if we had tutorials run in CI :(

Comment (dylanhmorris) [reply demos/hosp_only_ww_model/wwinference.Rmd]: Created #191

**Review by dylanhmorris — APPROVED** [review]

**Pull request merged**